free-2.md

```diff
@@ -58,12 +68,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Perhaps surprisingly, language will be a big part of this project. For the system aspect, the language can be "translated" into HDL code, which should be fairly straightforward if you know what your doing. This will definitely be the case when high-level components are simply just cookie-cutter code in the HDL.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+To fully support all the possibilities built in to a HDL would probably be pretty overwhelming. It might be easier to stick to the basics and make sure you can do that really well. That way it could at least be a language for beginners.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+There is definitely a need for a better HDL and this would be a good way to start. If successful, a project like this could save so many people's time. However, there are going to be a lot of hard decisions to make about the language and it will be tricky to support a lot of advanced HDL features. Automating stuff like state machines could turn into it's own DSL as they are so complicated.
```

Comment:
I am curious about some of these hard decisions - the only thing close to an HDL that i'm familiar with is logisim, so I'm curious what kinds of difficult decisions you would have to make in creating a better HDL.